packages/app/src/debug/DebugContainer.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div>
+    <div
+      v-if="loginConnectStore.user.isLoggedIn && loginConnectStore.project.isProjectConnected && run"
+    >
+      <DebugPageHeader
+        :gql="run"
+        :commits-ahead="0"
```

Comment:
Temporary placeholder value?

Reply:
Yes.  Will be replaced with the actual lookup in the final IATR M0 issue.